packages/ia-topnav/src/media-menu.js

```diff
@@ -75,6 +76,25 @@ class MediaMenu extends LitElement {
     this.config = {};
     this.openMenu = '';
     this.selectedMenuOption = '';
+    this.currentTab = {};
+  }
+
+  updated(props) {
+    if (props.has('currentTab')) {
+      const mediaButtons = Array.from(this.shadowRoot.querySelectorAll('media-button'));
+
+      mediaButtons.map((button, index) => {
+        const linkItem = button.shadowRoot.querySelector('a.menu-item');
+        if (linkItem) {
+          if (linkItem.classList.contains(`${this.selectedMenuOption}`)) {
+            linkItem.classList.remove('selected');
+            linkItem.blur();
+
+            mediaButtons[this.currentTab.moveTo === 'next' ? index + 1 : index - 1].shadowRoot.querySelector('a.menu-item').focus();
```

Comment:
Just for ease of reading & avoiding overly long lines, could we do something like:

```suggestion
            const newFocusIndex = this.currentTab.moveTo === 'next' ? index + 1 : index - 1;
            mediaButtons[newFocusIndex].shadowRoot.querySelector('a.menu-item').focus();
```